include/swift/AST/DiagnosticEngine.h

```diff
@@ -781,7 +781,7 @@ namespace swift {
 
   /// Class responsible for formatting diagnostics and presenting them
   /// to the user.
-  class DiagnosticEngine {
+  class __attribute__((swift_attr("import_as_ref"))) DiagnosticEngine {
```

Comment:
Could this be behind a preprocessor macro?

Reply:
Yeap, created a macro for this.